### PR TITLE
DCS-604 Prevent logging of post requests in licences

### PR DESCRIPTION
**Pull request by bell-pa**
Stop using the Winston Console transport when application insights is used.

### CODE REVIEW

File: server/app.js [hunk 1]
```diff
@@ -200,29 +201,31 @@ module.exports = function createApp({
...
-  // Request logging
-  app.use(
-    expressWinston.logger({
-      winstonInstance: logger,
-      meta: true,
-      dynamicMeta(req, res) {
-        const meta = {
-          userEmail: req.user ? req.user.username : null,
-          requestId: req.id,
-          sessionTag: req.user ? req.user.sessionTag : null,
```
Comment (andrewrlee) [inline server/app.js:212]: don't think this could have been working?!

**Review by andrewrlee — APPROVED** [review]
LGTM

File: server/app.js (hunk 1 above)
Comment (bell-pa) [reply server/app.js:212]: I've removed it completely and replaced with middleware that logs path, username and role as custom dimensions.

**Review by andrewrlee — APPROVED** [review]
LGTM

**Pull request merged**